nextflow.config

```diff
@@ -18,15 +18,14 @@ params {
     igenomes_base = 's3://ngi-igenomes/igenomes/'
     igenomes_ignore = false
     genomes_base = null // Disabled by default
-    save_reference = null // Built references not saved
+    save_reference = false // Built references not saved
```

Comment:
sometimes it's false and sometimes null, what is the rule here

Reply:
So I choose to use `false` for boolean, and `null` for every other types

Reply:
ok, makes sense